modules/apps/roles/roles-admin-web/src/main/resources/META-INF/resources/init.jsp

```diff
@@ -237,7 +238,7 @@ private StringBundler _getResourceHtmlId(String resource) {
 private boolean _isImpliedRole(Role role) {
 	String name = role.getName();
 
-	if (name.equals(RoleConstants.GUEST) || name.equals(RoleConstants.ORGANIZATION_USER) || name.equals(RoleConstants.OWNER) || name.equals(RoleConstants.SITE_MEMBER) || name.equals(RoleConstants.USER)) {
+	if (name.equals(RoleConstants.GUEST) || name.equals(RoleConstants.ORGANIZATION_USER) || name.equals(RoleConstants.OWNER) || name.equals(RoleConstants.SITE_MEMBER) || name.equals(RoleConstants.USER) || name.equals(AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MEMBER)) {
```

Comment:
This no longer applies since the change made in https://github.com/liferay/liferay-portal/commit/08054f6c7df4c465b904f255af5d4c47bfaf2c58. I've updated it to use the AccountRoleTypeContributor, please see https://github.com/pei-jung/liferay-portal/pull/620/commits/69442eb44c6335f3385b321fd7813ed4c8b386ab